Makefile

```diff
@@ -29,6 +29,10 @@ K3D_CLUSTER_NAME      ?= k3s-default
 KUBE_NAMESPACE        ?= argo
 MANAGED_NAMESPACE     ?= $(KUBE_NAMESPACE)
 
+E2E_TIMEOUT           ?= 1m
+E2E_PARALLEL          ?= 10
```

Comment:
Parallelism?

Reply:
This is for parallel tests.

`go test -failfast -v -timeout $(E2E_SUITE_TIMEOUT) -count 1 --tags api,cli,cron,executor,examples,functional,plugins -parallel $(E2E_PARALLEL) ./test/e2e  -run='.*/$*'`